{
    "expected": [
        {
            "process": {
                "pid": 54053,
                "thread": {
                    "id": 0
                }
            },
            "nginx": {
                "error": {
                    "connection_id": 1,
                    "time": "2016/10/25 14:49:34"
                }
            },
            "log": {
                "level": "error"
            },
            "message": "open() \"/usr/local/Cellar/nginx/1.10.2_1/html/favicon.ico\" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: \"GET /favicon.ico HTTP/1.1\", host: \"localhost:8080\", referrer: \"http://localhost:8080/\"",
            "error": {
                "message": "field [@timestamp] doesn't exist"
            }
        },
        {
            "process": {
                "pid": 54053,
                "thread": {
                    "id": 0
                }
            },
            "nginx": {
                "error": {
                    "connection_id": 3,
                    "time": "2016/10/25 14:50:44"
                }
            },
            "log": {
                "level": "error"
            },
            "message": "open() \"/usr/local/Cellar/nginx/1.10.2_1/html/adsasd\" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: \"GET /adsasd HTTP/1.1\", host: \"localhost:8080\"",
            "error": {
                "message": "field [@timestamp] doesn't exist"
            }
        },
        {
            "process": {
                "pid": 205860,
                "thread": {
                    "id": 205860
                }
            },
            "nginx": {
                "error": {
                    "connection_id": 180289,
                    "time": "2019/10/30 23:26:34"
                }
            },
            "log": {
                "level": "error"
            },
            "message": "FastCGI sent in stderr: \"PHP message: PHP Warning:  Declaration of FEE_Field_Terms::wrap($content, $taxonomy, $before, $sep, $after) should be compatible with FEE_Field_Post::wrap($content, $post_id = 0) in /var/www/xxx/web/wp-content/plugins/front-end-editor/php/fields/post.php on line 0",
            "error": {
                "message": "field [@timestamp] doesn't exist"
            }
        },
        {
            "message": "PHP message: PHP Warning:  Declaration of FEE_Field_Tags::wrap($content, $before, $sep, $after) should be compatible with FEE_Field_Terms::wrap($content, $taxonomy, $before, $sep, $after) in /var/www/xxx/web/wp-content/plugins/front-end-editor/php/fields/post.php on line 0",
            "error": {
                "message": "Provided Grok expressions do not match field value: [PHP message: PHP Warning:  Declaration of FEE_Field_Tags::wrap($content, $before, $sep, $after) should be compatible with FEE_Field_Terms::wrap($content, $taxonomy, $before, $sep, $after) in /var/www/xxx/web/wp-content/plugins/front-end-editor/php/fields/post.php on line 0]"
            }
        },
        {
            "message": "PHP message: PHP Warning:  Declaration of FEE_Field_Category::wrap($content, $sep, $parents) should be compatible with FEE_Field_Terms::wrap($content, $taxonomy, $before, $sep, $after) in /var/www/xxx/web/wp-content/plugins/front-end-editor/php/fields/post.php on line 0",
            "error": {
                "message": "Provided Grok expressions do not match field value: [PHP message: PHP Warning:  Declaration of FEE_Field_Category::wrap($content, $sep, $parents) should be compatible with FEE_Field_Terms::wrap($content, $taxonomy, $before, $sep, $after) in /var/www/xxx/web/wp-content/plugins/front-end-editor/php/fields/post.php on line 0]"
            }
        },
        {
            "process": {
                "pid": 54053,
                "thread": {
                    "id": 0
                }
            },
            "nginx": {
                "error": {
                    "connection_id": 3,
                    "time": "2019/11/05 14:50:44"
                }
            },
            "log": {
                "level": "error"
            },
            "message": "open() \"/usr/local/Cellar/nginx/1.10.2_1/html/adsasd\" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: \"GET /pysio HTTP/1.1\", host: \"localhost:8080\"",
            "error": {
                "message": "field [@timestamp] doesn't exist"
            }
        }
    ]
}